sdk/security/keyvault/azadmin/backup/client.go

```diff
@@ -110,7 +110,7 @@ func (client *Client) BeginFullRestore(ctx context.Context, restoreBlobDetails R
 // Generated from API version 7.5-preview.1
 func (client *Client) fullRestore(ctx context.Context, restoreBlobDetails RestoreOperationParameters, options *BeginFullRestoreOptions) (*http.Response, error) {
```

Comment:
Also, is this code still up to date with the new code generator? Because of the custom poller, I moved this code to custom_client.go. It might not follow the format of the new generated code

Reply:
I can regenerate and update it if need be

Reply:
Looking...

Reply:
Good catch! We just need to propagate the client's `tracing.Tracer` to the `runtime.Poller[T]`, no need for the custom polling handler to have it.